document/CoreFormat.mdk

```diff
@@ -1255,19 +1414,19 @@ If none of those strings accurately describes the construct, `kind` may contain
 
 ### General { #annotatedCodeLocation-General }
 
-An `annotatedCodeLocation` object is an element of an `executionFlow` object (&sect;[#result-executionFlows]),
+An `annotatedCodeLocation` object is an element of an `codeFlow` object (&sect;[#result-codeFlows]),
 a `stack` object (&sect;[#result-callStacks]),
 or the `relatedLocations` array (&sect;[#result-relatedLocations]).
 
 ### `physicalLocation` property { #annotatedCodeLocation-physicalLocation }
 
 An `annotatedCodeLocation` object must contain a property named `physicalLocation` whose value is a `physicalLocation` object
-(&sect;[#physicalLocation-info]) that specifies the code location to which the `annotatedCodeLocation` object refers.
+(&sect;[#physicalLocation]) that specifies the code location to which the `annotatedCodeLocation` object refers.
```

Comment:
this is really a file location, right? in other words, a class definition is a code location, which we might be able to provide a file location for (a physical location) or a logical (the namespace and class name).